content/lessons/chapter-5/outro-1.tsx

```diff
@@ -4,8 +4,8 @@ import { useTranslations } from 'hooks'
 import { ChapterEnd } from 'ui'
 
 export const metadata = {
-  title: 'chapter_four.outro_one.title',
-  image: '/assets/images/chapter-4-finish.jpg',
+  title: 'chapter_five.outro_one.title',
+  image: '/assets/images/chapter-5-outro.jpg',
   theme: 'bg-transparent/20',
   secondaryTheme: 'bg-[#472B26]',
   gradientTheme: 'from-[#472B26] via-[rgba(71, 43, 38, 0)]',
```

Comment:
Also, for mobile we want to change the background to be this color so that it matches the image closer `#2C3E59`